_pages/tutorial/2x.md

```diff
@@ -34,10 +34,26 @@ This tutorial uses QuickStart, a simple User Interface that you can run locally
 
 Before you can run the hub, you will need to have some some software installed.
 
-- [Oracle's Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) _(We have not tested with OpenJDK)_{:.smaller}
-- [MarkLogic 9](https://developer.marklogic.com/products) _(Installed and initialized)_{:.smaller}
+- [Oracle's Java 8 JRE](http://www.oracle.com/technetwork/java/javase/downloads/index.html) _(We have not tested with OpenJDK)_{:.smaller}  
+  **Not sure which version you have?**  
+  Try running this from a command line:
+  <pre class="cmdline">
+  java -version
+  </pre>
+- [MarkLogic 9](https://developer.marklogic.com/products) _(Installed and initialized)_{:.smaller}  
+  **Not sure?**  
+  Open your web browser to [http://localhost:8001](http://localhost:8001)  
+  After you logon look at the top left corner for the version info:  
+  ![Check ML Version]({{site.baseurl}}/images/2x/ml-version-check.png)
 - A Modern Browser. _(Chrome or FireFox work best. Use IE at your own risk.)_{:.smaller}
 
+## Common Concerns
+1. **I have a MarkLogic instance but it already has awesome stuff in it. Will this tutorial mess that up?**  
```

Comment:
While the database and forest names are unlikely to conflict, I think it's worth mentioning the ports that the tutorial will use.